install/ossm-cert-manager-assembly.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: ASSEMBLY
+[id="ossm-cert-manager"]
+= {SMProductName} and cert-manager
+include::_attributes/common-attributes.adoc[]
+:context: ossm-cert-manager
+//Working IA. The IA for 3.x is influx. Putting in "Installing" for now as most of the content relates to installing cert-manager, istio-csr, and then creating and installing the Istio resource.
+//TP1 content. Likely structure, titles, content, etc will change for GA
+//User must install cert-manager before they create and install an Istio resource. However, the spec.updateStrategy they decide on for the Istio resource dictates specific steps istio-csr and cert-manager, esp in regards to how istio-csr and cert-manager are updated.
+
+toc::[]
+
+The cert-manager tool is a solution for X.509 certificate management on Kubernetes. It delivers a unified API to integrate applications with private or public key infrastructure (PKI), such as Vault, Google Cloud Certificate Authority Service, Let's Encrypt, and other providers.
+
+It must be installed before you create and install your `Istio` resource.
+
+include::modules/ossm-about-cert-manager.adoc[leveloffset=+1]
+include::modules/ossm-install-cert-manager.adoc[leveloffset=+1]
+
+.Next steps
+To install `istio-csr`, you must follow the `istio-csr` installation instructions for the type of update strategy you want. By default, `spec.updateStrategy` is set to `InPlace` when you create and install your `Istio` resource. You create and install your `Istio` resource after you install `istio-csr`.
+
+* xref:../../ossm-cert-manager-istio-csr-inplace-update-strategy.adoc[istio-csr installation: InPlace update strategy]
+* xref:../../ossm-cert-manager-istio-csr-revisionbased-strategy.adoc[istio-csr installation: RevisionBased update strategy]
```

Comment:
Will these links work once this PR is merged? Get 404 now in the preview.

Reply:
Yes. As I stated in the Slack message, 404s are expected because this content has not been published.